style.css

```diff
@@ -0,0 +1,1427 @@
+  margin-top: 2px;
+  justify-content: space-between;
+  display: flex;
+}
+
+.apps-p2-text,
+.tech-p2-text,
+.gear-p2-text,
+.creative-p2-text,
+.podium-p2-text,
+.insights-p2-text,
+.launch-p2-text,
+.hard-fork-p2-text,
+.distract-p2-text {
+  padding: 0 5px 0 5px;
+  display: flex;
+  flex-direction: column;
+  color: #f42;
+}
+
+.apps-p2-text h5,
+.tech-p2-text h5,
+.gear-p2-text h5,
+.creative-p2-text h5,
+.podium-p2-text h5,
+.insights-p2-text h5,
+.launch-p2-text h5,
+.hard-fork-p2-text h5,
+.distract-p2-text h5 {
+  font-weight: 200;
+  margin-bottom: 0;
+}
+
+.hard-fork ul,
+.gear ul,
+.tech ul,
+.creative ul,
+.podium ul,
+.insights ul,
+.launch ul,
+.apps ul,
+.distract ul {
+  list-style: none;
+  padding-left: 1em;
+}
+
+.apps ul li,
+.gear ul li,
+.tech ul li,
+.creative ul li,
+.podium ul li,
+.insights ul li,
+.launch ul li,
+.hard-fork ul li,
+.distract ul li {
+  border-bottom: 1px solid rgba(0, 0, 0, 0.1);
+  font-size: 13px;
+  font-weight: 400;
+  line-height: 1.5;
+  padding: 2px 0 4px 0;
+  color: #555;
+}
+
+.hard-fork ul li::before {
+  color: #f42;
+  font-weight: bold;
+  display: inline-block;
+  width: 1em;
+  margin-left: -1em;
+  content: "\2022";
+}
+
+.apps ul li::before {
+  color: #f42;
+  font-weight: bold;
+  display: inline-block;
+  width: 1em;
+  margin-left: -1em;
+  content: "\2022";
+}
+
+.gear ul li::before {
+  color: #f42;
+  font-weight: bold;
+  display: inline-block;
+  width: 1em;
+  margin-left: -1em;
+  content: "\2022";
+}
+
+.tech ul li::before {
+  color: #f42;
+  font-weight: bold;
+  display: inline-block;
+  width: 1em;
+  margin-left: -1em;
+  content: "\2022";
+}
+
+.creative ul li::before {
+  color: #f42;
+  font-weight: bold;
+  display: inline-block;
+  width: 1em;
+  margin-left: -1em;
+  content: "\2022";
+}
+
+.podium ul li::before {
+  color: #f42;
+  font-weight: bold;
+  display: inline-block;
+  width: 1em;
+  margin-left: -1em;
+  content: "\2022";
+}
+
+.insights ul li::before {
+  color: #f42;
+  font-weight: bold;
+  display: inline-block;
+  width: 1em;
+  margin-left: -1em;
+  content: "\2022";
+}
+
+.launch ul li::before {
+  color: #f42;
+  font-weight: bold;
+  display: inline-block;
+  width: 1em;
+  margin-left: -1em;
+  content: "\2022";
+}
+
+.distract ul li::before {
+  color: #f42;
+  font-weight: bold;
+  display: inline-block;
+  width: 1em;
+  margin-left: -1em;
+  content: "\2022";
+}
+
+.img-great-mid {
+  height: 96px;
+  width: 96px;
+}
+
+.ld-head {
+  width: 86%;
+  height: auto;
+  margin: auto;
+  justify-content: space-between;
+  padding-bottom: 15px;
+  display: flex;
+}
+
+.ld-image {
+  width: 100%;
+  height: auto;
+}
+
+.left-ld-head {
+  text-decoration: none;
+  font-size: 28px;
+  letter-spacing: -0.015em;
+  font-weight: 700;
+  color: #f42;
+}
+
+.right-ld-head {
+  text-decoration: none;
+  font-size: 18px;
+  color: #f42;
+}
+
+.left-ld-head:hover {
+  color: #000;
+}
+
+.right-ld-head:hover {
+  color: #000;
+}
```

Comment:
## Aim to Stay `DRY`
- [ ] The lines of code on line `#715` & `#719` appear to be similar and make you seem like you are repeating yourself. Would you mind merging them so as to not appear redundant?